### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by nc6**
Each commit can be reviewed independently.

### CODE REVIEW

File: libs/cardano-ledger-core/src/Cardano/Ledger/Hashes.hs [hunk 1]
```diff
@@ -68,7 +68,7 @@ data EraIndependentScriptIntegrity
 
 type EraIndependentWitnessPPData = EraIndependentScriptIntegrity
 
--- $scriptHash
+-- scriptHash
```
Comment (lehins) [inline libs/cardano-ledger-core/src/Cardano/Ledger/Hashes.hs]: This was meant to be documentation for the header file. See `$scriptHash` anchor in the exports section. I think it would make sense to delete it from there too. A better alternative would be to revert this change and some haddock here.

**Review by lehins — APPROVED** [review]
Just one comment, otherwise looks great!

**Comment by lehins** [comment]
I think you also need to update ormolu in github actions https://github.com/input-output-hk/cardano-ledger/blob/4347e54b2827090e898702ccfb93cc731b4a1909/.github/workflows/haskell.yml#L145

File: libs/cardano-ledger-core/src/Cardano/Ledger/Hashes.hs (hunk 1 above)
Comment (nc6) [reply libs/cardano-ledger-core/src/Cardano/Ledger/Hashes.hs]: I went to add haddock, but there's really nothing to say, so I've just removed it.

**Review by JaredCorduan — APPROVED** [review]
thanks @nc6 !

**Comment by nc6** [comment]
Turns out we don't use the nix specified version in CI, so I've downbumped it to the 3.1.0 release
